src/main/java/uk/gov/hmcts/reform/civil/service/citizenui/HelpWithFeesForTabService.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.civil.service.citizenui;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.enums.FeeType;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.model.citizenui.HelpWithFeesDetails;
+import uk.gov.hmcts.reform.civil.model.citizenui.HelpWithFeesForTab;
+import uk.gov.hmcts.reform.civil.utils.MonetaryConversions;
+
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class HelpWithFeesForTabService {
+
+    private String hearingFee = "Hearing Fee";
+    private String claimIssuedFee = "Claim Fee";
+
+    public void setUpHelpWithFeeTab(CaseData.CaseDataBuilder caseDataBuilder) {
+        CaseData caseData = caseDataBuilder.build();
+        if (FeeType.CLAIMISSUED == caseData.getHwfFeeType()) {
+            caseDataBuilder.claimIssuedHwfForTab(setUpClaimIssuedHelpWithFeeTab(caseData));
+        } else if (FeeType.HEARING == caseData.getHwfFeeType()) {
+            caseDataBuilder.hearingHwfForTab(setUpHearingHelpWithFeeTab(caseData));
+        }
+    }
+
+    private HelpWithFeesForTab setUpHearingHelpWithFeeTab(CaseData caseData) {
+        return HelpWithFeesForTab.builder()
+            .remissionAmount(Optional.ofNullable(caseData.getHearingHwfDetails())
```

Comment:
Optional.ofNullable(caseData.getHearingHwfDetails())
                                 .map(HelpWithFeesDetails::getRemissionAmount)
                                .map(amount -> MonetaryConversions.penniesToPounds(amount)
                                 orElse(ZERO)
Default to Zero instead of null

Reply:
The issue is, if we set the 0, the related field will show when created. The AC mentioned that when created HWF without any remission selected, the related field will not shown.